account_draft_invoice_print/README.rst

```diff
@@ -27,6 +27,7 @@ Contributors
 
```

Comment:
Can you adapt the README file according the template ?

https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst